src/patterns/data/default/entity_attribute_partOf.tsv

```diff
@@ -0,0 +1,2 @@
+defined_class	defined_class_name	entity	entity_name	attribute	attribute_name
+OBA:VT0000074	neurocranium morphology trait	UBERON:0001703	neurocranium	PATO:0000051	morphology
```

Comment:
The [neurocranium morphology trait](http://purl.obolibrary.org/obo/OBA_VT0000074) is quite uninformative as a test case, because currently there are not any OBA morphology traits of neurocranium parts. 
[skull morphology trait](http://purl.obolibrary.org/obo/OBA_VT0000438) would be a better test case here for a grouping term.
See the figure here: https://github.com/obophenotype/bio-attribute-ontology/issues/139#issuecomment-1331284019